src/screens/ProfileScreen.js

```diff
@@ -27,13 +42,7 @@ const ProfileScreen = () => (
       <Data />
     </ScrollView>
     <Logout>
-      <View style={styles.logoutContainer}>
-        <TouchableOpacity style={styles.logout}>
```

Comment:
it seems like it would be more direct and clear to enclose `TouchableOpacity` with the `Logout` component (it would be functionally the same as what is here?)

see commit `41eac0bff85` of `fix-logout`


```
    <View style={styles.logoutContainer}>
      <Logout>
        <TouchableOpacity style={styles.logout}>
          <Text style={styles.logoutText}>
            Logout
          </Text>
        </TouchableOpacity>
      </Logout>
    </View>
```